docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Knative application under the strict authorization policy"
+weight: 25
+type: "docs"
+---
+
+When you deployed app to Knative Serving, serving system pods such as activator and autoscaler access to your app.
+Hence, you have to allow the requests to your app when you configure security features such as istio authorization policy.
+
+> Tip: This example assumes that your application enabled istio sidecar injection.
+>
+> ```
+> $ kubectl create namespace serving-tests
+> $ kubectl label namespace serving-tests istio-injection=enabled
+> ```
+> The following policy example does not work without sidecar injection.
+
+For example, the following authorization policy denies all requests to workloads in namespace serving-tests.
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: serving-tests
+spec:
+  {}
+EOF
+```
+
+Then, the following policy allows the request to `/` for your application.
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-app
+  namespace: serving-tests
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        paths:
+        - /     # The path for your application.
+EOF
+```
+
+It generally works with Kubernetes application, but it does not work with Knative application.
```

Comment:
Not sure why we show this either - we should document what _does_ work on Knative, not what doesn't?